src/index.js

```diff
@@ -46,7 +48,7 @@ function InfiniteLoading({
             >
               {({ index, style }) => (
                 <div style={style}>
-                  {children[index]}
+                  {children[index] != null ? children[index] : placeholder}
```

Comment:
I was thinking if we do:
```
{children[index] || placeholder}
```
it would work and fallback to `placeholder` if `children[index]` was empty. But I added the conditional above in case that's preferred.

Reply:
This looks cleaner and I think it would work but I'm not sure
```
{children[index] || placeholder}
```

Pretty sure this will work, but doesn't look as good
```
{children[index] != null ? children[index] : placeholder}
```

Reply:
If you try to access an index that doesn't exist in an array it returns undefined. This is different from null so you will try to display `children[index]`. Is that right?

Reply:
I believe the `!=` operator will check if null or undefined
https://stackoverflow.com/a/2672411/9830506

Reply:
It is right, didn't know about it 👌